charts/datadog/templates/cluster-agent-rbac.yaml

```diff
@@ -168,6 +168,13 @@ rules:
   - list
   - get
   - watch
+- apiGroups:
+  - autoscaling.k8s.io
+  resources:
+  - verticalpodautoscalers
+  verbs:
+  - list
+  - watch
```

Comment:
the VPA RBAC are added to the `cluster-agent` but how it works if Orchestrator Explorer collection is configure to run in a cluster-check-runner?

Reply:
@clamoriniere for now only the operator supports the runner setup "out of the box", on the chart we want to keep it simple for now